metabase/metabase
# Show inactive tables in data browser

Epic: https://github.com/metabase/metabase/issues/18675

Adds disabled cards in loading state for tables that are not synced yet.

<img width="1053" alt="Screen Shot 2021-10-27 at 12 40 15 pm" src="https://user-images.githubusercontent.com/8542534/139041558-385209fb-481d-4087-a18d-6634e0039a38.png">

**How to test**
As the BE for this PR is not ready we would need to rely on mocked data:
1. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L19
2. Set `initial_sync` to false https://github.com/metabase/metabase/blob/c0b5c697d1c5c814ec35a50ed2696617ea172d4a/frontend/src/metabase/schema.js#L53
3. Navigate to `http://localhost:3000/browse`
4. Select Sample Dataset
5. You should be able to see disabled table cards
6. Check that BE is polled for new statuses
7. Check that when all tables have `initial_sync: true`, BE is not polled

Please note that it should be OK to merge this PR before BE changes as with `initial_sync` set statically to `true` users won't see FE changes.
